### PR TITLE
MobileSkeleton Project

### PR DESCRIPTION
**Pull request by mstanley-me**
microverse: Mobile Skeleton Project

In this project I

Added HTML file contain a header and a section
Added css file to
Added a README file
Added a .gitignore file
Added an images folder

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                <a href="#index"><img src="./images/xDev.png" alt="logo"></a>
+            </div>
+            <div id="menu">
+                <ul>
+                    <li class="li"><a href="#works-content-box">Portfolio</a></li>
+                    <li class="li"><a href="#about-content-box">About</a></li>
+                    <li class="li"><a href="#contact-content-box">Contact</a></li>
+                    <li id="menu-box-model">
+                        <i class="fa fa-bars"></i>
+                    </li>
```
Comment (aliabbani) [inline index.html:26]: Please make sure that the hamburger icon has a nav tag. You can put it outside the `ul` tag and change the `li` to `nav` 😊

**Review by aliabbani — CHANGES_REQUESTED** [review]
Hi @mstanley-me ✋, 

Good job so far, you really did a great job!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Keep being Awesome!**
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 1 above)
Comment (mstanley-me) [reply index.html:26]: Thank you so much!

**Review by brytebee — APPROVED** [review]
Hi @mstanley-me👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**